src/web/routes/application/middleware/handle-path-request/handle-path-request.js

```diff
@@ -3,11 +3,12 @@ const { stateMachine, actions, states } = require('../../common/state-machine')
 const { isPathInApplicationFlow, stepNotNavigable } = require('./predicates')
 const { getPathsInSequence } = require('./selectors')
 
-const { IS_PATH_ALLOWED, GET_NEXT_ALLOWED_PATH } = actions
+const { COMPLETED } = states
+const { IS_PATH_ALLOWED, GET_NEXT_ALLOWED_PATH, SET_NEXT_ALLOWED_PATH } = actions
 
 const middleware = (config, pathsInSequence, step) => (req, res, next) => {
   // Destroy the session on navigating away from CONFIRM_URL
-  if (stateMachine.getState(req) === states.COMPLETED && req.path !== CONFIRM_URL) {
+  if (stateMachine.getState(req) === COMPLETED && req.path !== CONFIRM_URL) {
```

Comment:
I think this could be refactored out into its own function for clarity.

Reply:
Agreed, but updating functions in this file is not really the focus of this PR. I might do this in a subsequent PR.